src/Common/PricingTable/PricingTable.jsx

```diff
@@ -80,6 +81,7 @@ function PricingTable({ items }) {
 
   return (
     <div className="pricing-table-container">
+      <PageTitle value="Pricing" color="light" />
```

Comment:
Personally... I think that this belongs in `IndividualService.jsx`. The component is the pricing table, whereas this is a header for the pricing table.

Reply:
ignore me, this is the pricing section, rather than table.